mythril/laser/ethereum/plugins/implementations/state_merge.py

```diff
@@ -0,0 +1,299 @@
+from copy import copy
```

Comment:
This is looking good.
What do you think about separating this into a few files & putting it in a separate dir? 
I feel like we could go either way with this one, kind of an border case.
Anyway, wat are your thoughts on this?

Possible separation:
```
implementations/state_merge:
  __init__ (the state merge class, that directs all the other stuff)
  merge_condition_check ( the merge checking logic )
  merge ( the merging logic )
```